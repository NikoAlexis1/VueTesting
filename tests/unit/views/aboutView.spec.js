import { shallowMount } from '@vue/test-utils'
import AboutView from '@/views/AboutView.vue'

describe('HomeView', () => {
    test('ValidaciÃ³n de match con el snapshot', () => {
        const wrapper = shallowMount(AboutView)
        expect(wrapper.html()).toMatchSnapshot()
    })
})