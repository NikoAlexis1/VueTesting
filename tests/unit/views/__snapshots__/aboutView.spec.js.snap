// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeView Validaci√≥n de match con el snapshot 1`] = `
<div>
  <h1>About</h1>
</div>
`;
